<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarGPS</class>
 <widget class="QWidget" name="CarGPS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CarGPS</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>731</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnMenu">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/menu.png</normaloff>:/menu.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(186, 189, 182);</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QLineEdit" name="txtSearch">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>600</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>40</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/search_2.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>起点</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="txtStart">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>231</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="txtEnd">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>10</y>
            <width>231</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>10</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>终点</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>600</x>
            <y>10</y>
            <width>52</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>801</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetMenu" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <widget class="QPushButton" name="btnDirectionMode">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>101</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导航</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnNearbyMode">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>101</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>19</pointsize>
        </font>
       </property>
       <property name="text">
        <string>附近</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWebEngineView" name="mapView" native="true"/>
    </item>
   </layout>
   <zorder>mapView</zorder>
   <zorder>widgetMenu</zorder>
  </widget>
  <widget class="QWidget" name="keywordWidget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>681</width>
     <height>311</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="keyWordList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(46, 52, 54);
border-right: 5px solid rgb(136, 138, 133);
border-bottom: 5px solid rgb(136, 138, 133);
border-bottom-right-radius: 70px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/return.png</normaloff>:/return.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-4</y>
     <width>811</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>keywordWidget</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
